### ISSUE TITLE
Fatal error on pulumi up

### ISSUE THREAD

**Issue by jamesstow** [pulumi/pulumi#4677]
Pulumi Version:   v2.2.1
Go Version:       go1.13.10
Go Compiler:      gc
Architecture:     amd64
Operating System: linux
Panic:            runtime error: index out of range [-1]
```
goroutine 1 [running]:
runtime/debug.Stack(0xc0019722e8, 0x16fcb40, 0xc0015a9740)
        /home/travis/.gimme/versions/go1.13.10.linux.amd64/src/runtime/debug/stack.go:24 +0x9d
main.panicHandler()
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/cmd/pulumi/main.go:29 +0x6e
panic(0x16fcb40, 0xc0015a9740)
        /home/travis/.gimme/versions/go1.13.10.linux.amd64/src/runtime/panic.go:679 +0x1b2
gopkg.in/AlecAivazis/survey%2ev1.(*Select).OnChange(0xc0011cb040, 0x0, 0x0, 0x0, 0x0, 0x10, 0x29d84c8, 0x0, 0x0, 0x0, ...)
        /home/travis/gopath/pkg/mod/gopkg.in/!alec!aivazis/survey.v1@v1.8.8/select.go:154 +0x6d5
gopkg.in/AlecAivazis/survey%2ev1.(*Select).Prompt(0xc0011cb040, 0x0, 0x0, 0x0, 0x0)
        /home/travis/gopath/pkg/mod/gopkg.in/!alec!aivazis/survey.v1@v1.8.8/select.go:234 +0x3cf
gopkg.in/AlecAivazis/survey%2ev1.Ask(0xc001972948, 0x1, 0x1, 0x14abf60, 0xc00157f6c0, 0x0, 0x0, 0x0, 0x179c260, 0x14e1401)
        /home/travis/gopath/pkg/mod/gopkg.in/!alec!aivazis/survey.v1@v1.8.8/survey.go:148 +0x187
gopkg.in/AlecAivazis/survey%2ev1.AskOne(0x1ba0100, 0xc0011cb040, 0x14abf60, 0xc00157f6c0, 0x0, 0x0, 0x0, 0x0, 0x42, 0x0)
        /home/travis/gopath/pkg/mod/gopkg.in/!alec!aivazis/survey.v1@v1.8.8/survey.go:94 +0xcd
github.com/pulumi/pulumi/pkg/v2/backend.confirmBeforeUpdating(0x17f3955, 0x6, 0x1bc8600, 0xc000121700, 0xc00156c000, 0x5b, 0x80, 0x29d84c8, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/backend/apply.go:168 +0x439
github.com/pulumi/pulumi/pkg/v2/backend.PreviewThenPrompt(0x1ba9640, 0xc00003e0d8, 0x17f3955, 0x6, 0x1bc8600, 0xc000121700, 0xc000121780, 0xc0010a6a00, 0x32, 0xc00103bc80, ...)
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/backend/apply.go:127 +0x25a
github.com/pulumi/pulumi/pkg/v2/backend.PreviewThenPromptThenExecute(0x1ba9640, 0xc00003e0d8, 0x17f3955, 0x6, 0x1bc8600, 0xc000121700, 0xc000121780, 0xc0010a6a00, 0x32, 0xc00103bc80, ...)
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/backend/apply.go:199 +0x1ac
github.com/pulumi/pulumi/pkg/v2/backend/httpstate.(*cloudBackend).Update(0xc00101fad0, 0x1ba9640, 0xc00003e0d8, 0x1bc8600, 0xc000121700, 0xc000121780, 0xc0010a6a00, 0x32, 0xc00103bc80, 0x29d84c8, ...)
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/backend/httpstate/backend.go:812 +0xe6
github.com/pulumi/pulumi/pkg/v2/backend.UpdateStack(0x1ba9640, 0xc00003e0d8, 0x1bc8600, 0xc000121700, 0xc000121780, 0xc0010a6a00, 0x32, 0xc00103bc80, 0x29d84c8, 0x0, ...)
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/backend/stack.go:82 +0xc4
github.com/pulumi/pulumi/pkg/v2/backend/httpstate.(*cloudStack).Update(0xc000121700, 0x1ba9640, 0xc00003e0d8, 0xc000121780, 0xc0010a6a00, 0x32, 0xc00103bc80, 0x29d84c8, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/backend/httpstate/stack.go:148 +0x96
main.newUpCmd.func1(0x29d84c8, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7fffffff, 0x0, 0x29d84c8, 0x0, ...)
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/cmd/pulumi/up.go:133 +0xb0e
main.newUpCmd.func3(0xc0001d3600, 0x29d84c8, 0x0, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/cmd/pulumi/up.go:372 +0x4e1
github.com/pulumi/pulumi/sdk/v2/go/common/util/cmdutil.RunResultFunc.func1(0xc0001d3600, 0x29d84c8, 0x0, 0x0)
        /home/travis/gopath/src/github.com/pulumi/pulumi/sdk/go/common/util/cmdutil/exit.go:112 +0x6b
github.com/spf13/cobra.(*Command).execute(0xc0001d3600, 0x29d84c8, 0x0, 0x0, 0xc0001d3600, 0x29d84c8)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001d3080, 0x0, 0x0, 0xc000225f20)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
        /home/travis/gopath/src/github.com/pulumi/pulumi/pkg/cmd/pulumi/main.go:48 +0x63
```

**Comment by jamesstow** [comment]
This happened after an up, while pressing down and up a few times on the confirmation dialog.
Terminal emulator is Konsole as was bugging (konsole issue, not pulumi) not showing the output

**Comment by stack72** [comment]
Hi @jamesstow 

I'm sorry you received this error - please can you describe the steps that you were doing that caused this? Was tis immediately after a `pulumi new` or is it an existing project? 

Thanks
Paul

**Comment by jamesstow** [comment]
not reproducable

**Comment by stack72** [comment]
Hi all

We believe this is closed via #4808

Sorry this wasn't closed out before

Paul
